src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -418,7 +418,11 @@ void ScatterUpdate::scatterNDUpdate(uint8_t *indices, uint8_t *update, uint8_t *
         size_t indicesOffset = tupleIdx * k;
         size_t dstOffset = 0;
         for (int i = 0; i < k; i++) {
-            size_t idxValue = getIndicesValue(indices, indicesOffset + i);
+            int64_t idxValue = getIndicesValue(indices, indicesOffset + i);
+            if (idxValue < 0) {
+                // Negative value for indices means counting backwards from the end.
+                idxValue += srcDataDim[0];
```

Comment:
```suggestion
                idxValue += srcDataDim[i];
```
negative value conversion should be applied to current axis.